src/main/java/com/redhat/devtools/lsp4ij/features/documentSymbol/DocumentSymbolData.java

```diff
@@ -76,6 +79,11 @@ public int getTextOffset() {
         return getClientFeatures().getDocumentSymbolFeature().getTextOffset(documentSymbol, psiFile);
     }
 
+    @Override
+    public TextRange getTextRange() {
+        return LSPIJUtils.toTextRange(documentSymbol.getRange(), LSPIJUtils.getDocument(psiFile.getVirtualFile()));
```

Comment:
I will store the text range is a field, since I suppose this method can be called several times.

Reply:
Done